### PR TITLE
R tftargets

### PR DESCRIPTION
**Pull request by khughitt**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-tftargets/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/slowkow/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e27ed65c283779b4dcda6e0704a37112a81bfedb198b3d7d036c1a80083670ca
+
+build:
+  number: 1
```
Comment (npavlovikj) [inline recipes/r-tftargets/meta.yaml]: The build number should be 0. You increment this only if a package with the same version is merged and is already available in the `bioconda` channel. Within a PR, regardless of the number of pushes, the build number should stay the same.

File: recipes/r-tftargets/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - https://github.com/slowkow/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: e27ed65c283779b4dcda6e0704a37112a81bfedb198b3d7d036c1a80083670ca
+
+build:
```
Comment (npavlovikj) [inline recipes/r-tftargets/meta.yaml]: Packages that don't have anything in the `compiler` section should be `noarch`, so please add `noarch: generic` here under the `build` section. Please read more about this here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages.

File: recipes/r-tftargets/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  number: 1
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
```
Comment (npavlovikj) [inline recipes/r-tftargets/meta.yaml]: This should be `host` instead, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section.

File: recipes/r-tftargets/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    - r-base
+
```
Comment (npavlovikj) [inline recipes/r-tftargets/meta.yaml]: Remove new line.

File: recipes/r-tftargets/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{% set name = 'tftargets' %}
+{% set version = '1.3' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
```
[the remark this answers is not in the document]
Comment (khughitt) [reply recipes/r-tftargets/meta.yaml]: Ah good catch! I used another github R package as template, and didn't notice the difference. I'll be a bit more careful next time.

File: recipes/r-tftargets/meta.yaml (hunk 2 above)
Comment (khughitt) [reply recipes/r-tftargets/meta.yaml]: Got it. Thanks for the reference!

File: recipes/r-tftargets/meta.yaml (hunk 4 above)
Comment (khughitt) [reply recipes/r-tftargets/meta.yaml]: Thanks for the thorough feedback and suggestions! It is much appreciated. I'll be sure to take a closer look into each of the config params next time around.

**Comment by npavlovikj** [comment]
@BiocondaBot recheck

**Comment by npavlovikj** [comment]
Thank you so much for adding the recipe and following the suggestions @khughitt ! Everyhing looks good to me now, so I went ahead and merged the recipe.

File: recipes/r-tftargets/meta.yaml (hunk 4 above)
Comment (npavlovikj) [reply recipes/r-tftargets/meta.yaml]: No worries, that is why we require 4-eyes review :)

**Pull request merged**